proposals/NNNN-webengine-projection-mode.md

```diff
@@ -0,0 +1,67 @@
+It extends [SDl-0240 - WebEngine support for SDL JavaScript](https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0240-sdl-js-pwa.md) with a projection mode (formally known as OpenHMI) that was introduce with [SDL-0031 Mobile Projection](https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0031-mobile-projection.md).
+
+## Motivation
+
+This chapter describes a feature made possible using a WebEngine. A WebEngine can come with a web page rendering component described as a WebView in this proposal. The proposal describes how apps based on a WebEngine can be presented not only using system templates but that can control the WebView using the app's document object.
+
+## Proposed solution
+
+The app HMI type `PROJECTION` should be enabled for in-vehicle apps. When apps with this HMI type are activated, the HMI should make the web page of this app visible on the screen. This web page will become the main window of the application. The window capabilities of this open main window will be empty except for the window ID and physical button capabilities. `Show` requests that address the main window won't have any effect on the HMI. If the app sends this request, Core should return an unsuccessful response with the result code `RESOURCE_NOT_AVAILABLE`. The info field should note that the app is registered with projection mode enabled.
+
+![Screenshot example of a web app](../assets/proposals/NNNN-webengine-projection-mode/web-app-example.jpg)
+
+> Example of a local web app presenting the user interface with the WebView.
+
+Widgets are still available and can be controlled using `Show` RPC. Any overlay like Alert, ChoiceSets, Slider etc. are also available to the application.
+
+##### Policy control
+
+The HMI type `PROJECTION` is policy controlled. On the policy server this HMI type can be added to the valid HMI type list per app ID. Only apps with permissions to use this HMI type would be allowed to register.
+
+##### User interface guidelines (Driver Distraction rules)
+
+With the current depth of this proposal, the HMI type should be used by 1st party OEM apps only. With future proposals and workshops the SDLC could open the HMI type to 3rd party by creating and defining proper driver distraction and user interface guidelines.
+
+At the time of this proposal being in review, a set of driver distraction rules are being created and proposed to enable 3rd party using the projection mode. The following bullet points list items that will be described further in the ruleset:
+
+-	minimum font size
+-	minimum contrast between font and background
+-	min/max brightness (for day and night mode)
+-	number of buttons
+-	minimum size of a button
+- No customized keyboard
+- No video playback (exceptions in standstill per countries)
+-	NHTSA related guidelines
+  o	Amount of text (button and text fields)
+  o	number of lines of text
+  o	Complexity of use cases (number of steps to fulfill a use case)

```

Comment:
These last three bullets are not formatting correctly in .md.  Can you please update?